app/controllers/admin/users_controller.rb

```diff
@@ -1,2 +1,29 @@
 class Admin::UsersController < Admin::BaseController
+  expose :user
+  expose :interests, from: :user
+  expose_decorated(:users, -> { User.all })
+
+  def create
+    user.password = 'secret'
```

Comment:
"[...] New users created by admin should have a password set to "secret". [...]"